.sauce/playwright.yml

```diff
@@ -1,26 +1,42 @@
 apiVersion: v1alpha
-# meta data to the test
-metadata:
-  name: Feature XYZ
-  tags:
-    - e2e
-    - release team
-    - other tag
-  build: "Build #$BUILD_ID in $BUILD_ENV"
-# Every file defined in this list will be bundled into a zip and
-# uploaded to Sauce Labs.
-files:
-  - ./tests/playwright/demo.test.js
-# Define a test runner image (e.g. an image to run WebdriverIO tests)
-# Like in Docker, these images can be developed as Open Source projects
-# and maintained by our teams, while at the same time, customers can
-# build their own images as well
+kind: playwright
+sauce:
+  region: us-west-1
+  concurrency: 1
+## Tunnel allows you to specify an existing sauce connect tunnel when running playwright inside the Sauce cloud.
+## This has no effect when running tests inside docker.
+#  tunnel:
+#    id: your_tunnel_id
+#    parent: parent_owner_of_tunnel # if applicable, specify the owner of the tunnel
+  metadata:
+    name: Testing Playwright Support
+    tags:
+      - e2e
+    build: "$BUILD_ID"
+docker:
+  # fileTransfer controls how test files are transferred to the docker container before tests are run (choice: mount|copy).
+  # `mount` will mount files and folders into the container. Changes to these files and folders will be reflected on the
+  # host as well (and vice versa). However, you may run into permission issues depending on your docker or host settings.
+  # In this case the usage of `copy` is advised. `copy` will simply copy files and folders into the container.
+  fileTransfer: mount
+  # image controls which images to be used for local testing. Change this value is you want to use a custom image.
+  # image:
+  #   name: saucelabs/stt-playwright-jest-node
```

Comment:
Could you update this one to `image: saucelabs/stt-playwright-jest-node:v1.7.1` please? :)

Reply:
Yep! That's what I just planed to do